extensions/basic/jruby-ext/src/main/java/com/purbon/RubyFoo.java

```diff
@@ -0,0 +1,61 @@
+package com.purbon;
+
+import org.jruby.*;
+import org.jruby.anno.JRubyMethod;
+import org.jruby.anno.JRubyModule;
+import org.jruby.runtime.ThreadContext;
+import org.jruby.runtime.builtin.IRubyObject;
+
+/**
+ * A basic module named RubyFoo, created for learning purposes
+ * Created by purbon on 24/08/15.
+ */
+@JRubyModule( name = "Foo")
+public class RubyFoo extends RubyObject {
+
+    public RubyFoo(Ruby ruby, RubyClass metaclass) {
+        super(ruby, metaclass);
+    }
+
+    /**
+     * Extension method with two names (aliases) that return an string). This is
+     * an static method.
+     * @param context The ThreadContext of the execution. (required)
+     * @param self A reference to self, aka like this in java but in ruby. (required)
+     * @return A RubyString.
+     */
+    @JRubyMethod( module = true, name = { "build_string", "new_string" } )
+    public static RubyString buildAnString(ThreadContext context, IRubyObject self) {
+        Ruby runtime = context.runtime;
+        return runtime.newString("This is a new String");
+    }
+
+    /**
+     * Extension method that add two numbers. ( NOTE: keep in mind we're in ruby, so
+     * parameters should be IRubyObjects, not directly numbers )
+     * @param context The ThreadContext in the execution.
+     * @param a A ruby object
+     * @param b A ruby object
+     * @return The outcome of doing a plus b.
+     */
+    @JRubyMethod(module = true, name = "add")
+    public IRubyObject add(ThreadContext context, IRubyObject a, IRubyObject b) {
```

Comment:
This method can probably go away right? It's already defined in the `RubyBar` class